test/unit/default_suite_tests.rb

```diff
@@ -33,18 +33,27 @@ class UnitTests < Assert::Context
     end
 
     should "know its tests-to-run atts" do
-      assert_equal 6, subject.tests_to_run.size
-      assert_equal 6, subject.tests_to_run_count
-      assert_kind_of Assert::Test, subject.tests.first
+      assert_equal @tests.size, subject.tests_to_run_count
       assert_true subject.tests_to_run?
 
       subject.clear_tests_to_run
 
-      assert_equal 0, subject.tests_to_run.size
       assert_equal 0, subject.tests_to_run_count
       assert_false subject.tests_to_run?
     end
 
+    should "find a test to run given a file line" do
+      test = @tests.sample
+      assert_same test, subject.find_test_to_run(test.file_line)
+    end
+
+    should "know its sorted tests tests to run" do
+      sorted_tests = subject.sorted_tests_to_run{ 1 }
+      assert_equal @tests.size, sorted_tests.size
+      assert_kind_of Assert::Test, subject.tests.first
```

Comment:
@kellyredding - Should this be `sorted_tests.first` instead of `subject.tests.first`?